test/common/run-tests

```diff
@@ -295,6 +296,8 @@ def main():
                         default=None, help="When using --machine, use this cockpit web address")
     parser.add_argument('--test-dir', default=testvm.TEST_DIR,
                         help="Directory in which to glob check-* files")
+    parser.add_argument('--exclude', action="append", default=[], metavar="SUBSTRING",
```

Comment:
hmm, not so convinced about "substring" and "match pattern". Those are two different things.
Substrings (which this PR uses) can be silently skipping some tests. Like if you have "testFoo" and "testFooBar" and you only want to skip "testFoo" it will also skip "testFooBar" and there is nothing you can do about it.
If it would be really a pattern matching, then "testFoo" would only skip "testFoo" and you would need to use something like "testFoo*" to also skip "testFooBar".
IMHO I think `--exclude` where you just specify exact matches would be better as it would not bring any unexpected surprises.

Reply:
Agreed. Fixed accordingly.